Assets/PatchKit Patcher/Resources/Languages/en.json

```diff
@@ -0,0 +1,57 @@
+{
+  "adding_new_files_0": "Adding new files...",
+  "adding_new_files_1": "Adding new files ({0}/{1})...",
+  "allocating_disk_space": "Allocating disk space...",
+  "applying_diffs_0": "Applying diffs...",
+  "applying_diffs_1": "Applying diffs ({0}/{1})...",
+  "applying_fixes": "Applying fixes...",
+  "cancel": "cancel",
+  "changelog": "Changelog",
+  "check_for_updates": "Check for updates",
+  "checking_version_integrity": "Checking version integrity...",
+  "confirm": "Confirm",
+  "dowload_speed_d": "day",
+  "dowload_speed_h": "hour",
+  "dowload_speed_kb": " KB/sec.",
+  "dowload_speed_m": "minute",
+  "dowload_speed_mb": " MB/sec.",
+  "dowload_speed_moment": "a moment",
+  "dowload_speed_s": "second",
+  "dowload_status_0": "Downloading package...",
+  "dowload_status_1": "Stalled...",
+  "downloading_fixes": "Downloading fixes...",
+  "enter_code": "Enter code",
+  "error": "An error has occured!",
+  "game_changelog": "GAME CHANGELOG",
+  "install": "Install",
+  "installing_0": "Installing...",
+  "installing_1": "Installing ({0}/{1})...",
+  "license": "Please entry license key",
+  "license_blocked": "Blocked license.",
+  "license_error": "Entred code is not valid!",
+  "license_invalid": "Invalid license.",
+  "license_service": "Service is unavailable.",
+  "no_internet_connection": "Please check your internet connection.",
+  "no_permissions": "Please check write permissions in application's directory.",
+  "non_launcher_execution": "Patcher has to be started using the launcher.",
+  "not_enough_disk_space": "Not enough disk space to install this application. Additional {0:0.00} GB of disk space is required.",
+  "other_error": "Unknown error, please try again. If the issue remains, please contact the support.",
+  "other_error_repiar": "Couldn't validate local files, please try again. If the issue remains, it could mean a disk issue.",
+  "play": "Play",
+  "progress_bar_0": "Connecting...",
+  "progress_bar_1": "Error...",
+  "published_at": "Published at:",
+  "removing_old_files_0": "Removing old files...",
+  "removing_old_files_1": "Removing old files ({0}/{1})...",
+  "retry": "Retry",
+  "show_changelog": "Changelog",
+  "status_0": "Loading data...",
+  "status_1": "Loading configuration...",
+  "status_2": "Starting application...",
+  "unablet_to_check_for_updates": "Unable to check for updates.",
+  "unarchiving_package_0": "Unarchiving package...",
```

Comment:
Please rename to `"unarchiving_package"`.

Reply:
done